### DISCUSSION TITLE
List of sections/items to include in website

### DISCUSSION THREAD

**Discussion by DesignrKnight** [dscnitrourkela/project-lime#1] [Ideas]
Let us use this discussion to make a draft of all things we need to add to the website.

**Comment by DesignrKnight** [comment]
Reference Link of NITRAA Global Summit Delhi
Explara Page: https://in.explara.com/e/b1b146ca4e37dcd3ff07ce64a8779d72

**Comment by DesignrKnight** [comment]
Reference Link of NITRAA Delhi
http://nitraadelhi.org/

**Comment by riteshsp2000** [comment]
The second website looks all good! Gives us a brief about the general structure and all.

Here is a list of requirements sent to us as of now.
1. Registration with name, batch-of passing out, single and/or with family
2. Till now registered with numbers (batch wise analytics)
3. Payment option with tariff - Early Bird Registration options.
4. Golden Memories (one picture from each batch)
5. Highlights of Year Long Activities
6. Souvenir and news bulletin Section (Press Section)
7. Highlights of sponsorship
8. Student Corner (New updates in the college)
9. Information about nearby hotels
10. Any other may be added later on


Not necessary:
1. Social Wall

**Comment by riteshsp2000** [comment]
Checkout these links as well for reference 

Convocation: https://nitrkl.ac.in/convocation/webpages/home.aspx
For Analytics: https://help.bevylabs.com/article/376-featured-attendees
Student Corner: https://nitrkl.ac.in/docs/PlacementBrochure/15072020165620245.pdf
